### ISSUE TITLE
Problem about installation

### ISSUE THREAD

**Issue by XMU-qcj** [ThibaultGROUEIX/ChamferDistancePytorch#19]
Thanks for your great work,
G++ error occurred when I entered python setup.py install.
`running install
running bdist_egg
running egg_info
writing chamfer_3D.egg-info/PKG-INFO
writing dependency_links to chamfer_3D.egg-info/dependency_links.txt
writing top-level names to chamfer_3D.egg-info/top_level.txt
listing git files failed - pretending there aren't any
reading manifest file 'chamfer_3D.egg-info/SOURCES.txt'
writing manifest file 'chamfer_3D.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_ext
building 'chamfer_3D' extension
creating /home/lq/New_p/VRCNet-main/chamfer3D/build
creating /home/lq/New_p/VRCNet-main/chamfer3D/build/temp.linux-x86_64-3.7
Emitting ninja build file /home/lq/New_p/VRCNet-main/chamfer3D/build/temp.linux-x86_64-3.7/build.ninja...
Compiling objects...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
1.10.2.git.kitware.jobserver-1
creating build/lib.linux-x86_64-3.7
g++ -pthread -shared -B /home/lq/anaconda3/envs/mvp/compiler_compat -L/home/lq/anaconda3/envs/mvp/lib -Wl,-rpath=/home/lq/anaconda3/envs/mvp/lib -Wl,--no-as-needed -Wl,--sysroot=/ /home/lq/New_p/VRCNet-main/chamfer3D/build/temp.linux-x86_64-3.7/chamfer_cuda.o /home/lq/New_p/VRCNet-main/chamfer3D/build/temp.linux-x86_64-3.7/chamfer3D.o -L/home/lq/anaconda3/envs/mvp/lib/python3.7/site-packages/torch/lib -L/usr/local/cuda-10.1/lib64 -lc10 -ltorch -ltorch_cpu -ltorch_python -lcudart -lc10_cuda -ltorch_cuda -o build/lib.linux-x86_64-3.7/chamfer_3D.cpython-37m-x86_64-linux-gnu.so
g++: error: /home/lq/New_p/VRCNet-main/chamfer3D/build/temp.linux-x86_64-3.7/chamfer_cuda.o: No such file or directory
g++: error: /home/lq/New_p/VRCNet-main/chamfer3D/build/temp.linux-x86_64-3.7/chamfer3D.o: No such file or directory
error: command 'g++' failed with exit status 1`

**Comment by ThibaultGROUEIX** [comment]
Hi @Unknownnnnnnn ,

Thanks for using the repo.
Are you sure this is the full log? 
I don't know how to help you from this alone unfortunately,

Best regards,
Thibault

**Comment by XMU-qcj** [comment]
> Hi @Unknownnnnnnn ,
> 
> Thanks for using the repo. Are you sure this is the full log? I don't know how to help you from this alone unfortunately,
> 
> Best regards, Thibault

Yes, this is all the error information,
Maybe the main error is these lines,
g++: error: /home/lq/New_p/VRCNet-main/chamfer3D/build/temp.linux-x86_64-3.7/chamfer_cuda.o: No such file or directory
g++: error: /home/lq/New_p/VRCNet-main/chamfer3D/build/temp.linux-x86_64-3.7/chamfer3D.o: No such file or directory
error: command 'g++' failed with exit status 1

**Comment by ThibaultGROUEIX** [comment]
Yes sure, but it does not say why these files are missing. Can you delete all of the build files (basically start from scratch) and rerun setup.py  ?

**Comment by XMU-qcj** [comment]
> Yes sure, but it does not say why these files are missing. Can you delete all of the build files (basically start from scratch) and rerun setup.py ?

I tried to delete build, but the error information is the same.

**Comment by XMU-qcj** [comment]
> Yes sure, but it does not say why these files are missing. Can you delete all of the build files (basically start from scratch) and rerun setup.py ?
Sorry to bother you again, but I still can't solve this problem.
I found that the files that I couldn't find were in another trash folder. Is this blocked from downloading by some system?
/home/lq/.local/share/Trash/files/build 5/temp.linux-x86_64-3.7/chamfer_cuda.o

**Comment by KevnDuta** [comment]
> > Yes sure, but it does not say why these files are missing. Can you delete all of the build files (basically start from scratch) and rerun setup.py ?
> > Sorry to bother you again, but I still can't solve this problem.
> > I found that the files that I couldn't find were in another trash folder. Is this blocked from downloading by some system?
> > /home/lq/.local/share/Trash/files/build 5/temp.linux-x86_64-3.7/chamfer_cuda.o

Hi, I meet the same problem, do you solve it?
